app/views/content/blog/no-two-days-are-the-same.md

```diff
@@ -0,0 +1,43 @@
+---
+title: No two days are the same
+date: "2020-01-13"
+author: Gavin McIntyre
+images:
+  q-gavin:
+    path: "media/images/content/blog/stories-gavin.jpg"
+    thumbnail_path: "media/images/content/blog/stories-gavin.jpg"
+description: |-
+  Assistant headteacher Gavin McIntyre shares his journey to become a teacher.
+keywords:
+  - life as a teacher
+  - day in the life
+  - classroom
+tags:
+  - why teach?
```

Comment:
Need to change this tag - doesn't work after 'Blog posts about' on the blog tag page

Reply:
It seems like it does for me - the path is `/blog/tag/why-teach` and it looks ok 🤔 

If the question mark had been included in the tag agree it would have totally broke!

Reply:
Ha! I wasn't clear - I meant it doesn't work grammatically not technically! Ellie is rewording.....